### DISCUSSION TITLE
How do I install SDK 9.0.100-rc.2.24468.2 ?

### DISCUSSION THREAD

**Discussion by voroninp** [dotnet/aspnetcore#58275] [Q&A] [answered]
`main` branch targets `9.0.100-rc.2.24468.2` in `global.json`. But how do I install it?
It's not officially available and I do not see it [in the table](https://github.com/dotnet/sdk/blob/main/documentation/package-table.md)

**Comment by martincostello** [comment, marked as answer]
That doesn't look like the version in main now, or earlier today.

Running the restore script should install it locally for the build to work for you as per the docs: https://github.com/dotnet/aspnetcore/blob/main/docs/BuildFromSource.md

**Reply by voroninp** [reply]
Hm, weird. Yesterday I ran restore, and could not build. Thanks.
